adops/setting-up-prebid-multi-format-in-dfp.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: page
+title: Setting up Prebid Multi-Format in DFP
+head_title: Setting up Prebid Multi-Format in DFP
+description: Setting up Prebid Multi-Format in DFP
+pid: 3
+hide: false
+top_nav_section: adops
+nav_section: tutorials
+---
+
+<div class="bs-docs-section" markdown="1">
+
+# Setting up Prebid Multi-Format in DFP
+{: .no_toc}
+
+This page shows how to set up your ad server so that you can serve multi-format ads.
+
+Multi-Format ads allow you to declare multiple media types on a single ad unit.  For example, you can set up one ad on the page that could show a banner, native, or outstream video ad, depending on which had the highest bid.
+
+{: .alert.alert-info :}
+For instructions on how to set up multi-format ads from the engineering side, see [Show Multi-Format Ads with Prebid.js]({{site.baseurl}}/dev-docs/show-multi-format-ads.html).
+
+* TOC
+{: toc }
+
+## Step 1. Add an Ad Unit
+
+In DFP, [create an ad unit](https://support.google.com/dfp_premium/answer/177203?hl=en).
+
+Decide what combination of formats will be permitted on the ad unit.  This will determine what sizes you allow to serve.  The ad unit's sizes must be configured properly to support the combination of formats that will be permitted.
+
+If your ad unit will support native ads, you must create a custom **Prebid Native Format** and **Prebid Native Style** as described in [Setting up Prebid Native in DFP][nativeAdSetup].
```

Comment:
Let's say: "If your ad unit will support native ads, you may want to create a custom **Prebid Native Format** and at least one **Prebid Native Style**.  Examples of each are given in..."

Reply:
Thanks, updated in 3c42b81